ext/comms_php.c

```diff
@@ -2,15 +2,16 @@
 
 #include "coms.h"
 #include "ddtrace.h"
-#include "logging.h"
+#include "configuration.h"
+#include <components/log/log.h>
 #include "mpack/mpack.h"
 
 ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
 
 bool ddtrace_send_traces_via_thread(size_t num_traces, char *payload, size_t payload_len) {
     if (!get_DD_TRACE_ENABLED()) {
         // If the tracer is set to drop all the spans, we do not signal an error.
-        ddtrace_log_debugf("Traces are dropped by PID %ld because tracing is disabled.", getpid());
+        LOG(Warn, "Traces are dropped by PID %ld because tracing is disabled.", getpid());
```

Comment:
If this is what the customer want, I assume we should not flood with warning.
Maybe logging once would be good as it seems a possibility

Reply:
This specific one does not matter much, it would only happen if one would use the manual API to send bytes directly to the agent. (which, I hope, nobody uses. It was used by us in the past. A candidate to deprecate.)